celery/backends/base.py

```diff
@@ -658,9 +658,9 @@ def _store_result(self, task_id, result, state,
                       traceback=None, request=None, **kwargs):
 
         if state in self.READY_STATES:
-            date_done = datetime.datetime.utcnow()
+            date_done = datetime.datetime.utcnow().isoformat()
         else:
-            date_done = None
+            date_done = ''
```

Comment:
What's wrong with None?

Reply:
Nothin wrong really. I thought it would be better to keep the value as a string instead of another object (null) when serializing. But if we want to standardize it to `None` I'm ok with it, too.